packages/config/config/devices/0x0438/4512757.json

```diff
@@ -0,0 +1,423 @@
+{
+	"manufacturer": "Namron",
+	"manufacturerId": "0x0438",
+	"label": "4512757",
+	"description": "16 A Thermostat - White",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x0108"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "4",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "ECO Mode",
+			"valueSize": 4,
+		},
+		{
+			"#": "8",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Window Check",
+			"valueSize": 4,
+			"defaultValue": 1,
+		},
+		{
+			"#": "10",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Frost",
+			"valueSize": 4,
+		},
+		{
+			"#": "23",
+			"label": "Temperature Unit",
+			"valueSize": 4,
+			"unit": "°C",
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "°C",
+					"value": 0
+				},
+				{
+					"label": "°F",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "27",
+			"label": "Temperature Calibration",
+			"valueSize": 4,
+			"minValue": -10,
+			"maxValue": 10,
+			"defaultValue": 0,
+		},
+		{
+			"#": "31",
+			"label": "Work Days Set",
+			"valueSize": 4,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "5_2",
+					"value": 0
+				},
+				{
+					"label": "6_1",
+					"value": 1
+				},
+				{
+					"label": "7_0",
+					"value": 2
+				},
+				{
+					"label": "0_7",
+					"value": 3
+				}
```

Comment:
These option labels are unclear. Can you replace them with something that one can understand at first glance?

Reply:
Replaced labels